packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java

```diff
@@ -1069,4 +1110,71 @@ public ResponseEntity<Void> cancelTask(@PathVariable("task-id") final UUID taskI
 		taskService.cancelTask(taskId);
 		return ResponseEntity.ok().build();
 	}
+
+	private TaskRequest getModelCardTask(DocumentAsset document, Model model, UUID projectId) {
```

Comment:
Should `getModelCardTask` and `getEnrichAMRTaskRequest` methods be in a service file instead of this controller?